docs/package manifests.md

```diff
@@ -0,0 +1,19 @@
+Tofino uses a two package manifest setup where node modules required to ship as-is with the
```

Comment:
This looks fine to me - maybe name the file `package-manifests` to be consistent with other files in the dir